### PR TITLE
Adding fluoriclogppka package

### PR DESCRIPTION
**Pull request by PetroPavliukk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by zklaus** [comment]
It's `pip check` that is failing, not detecting `rdkit`. Since the conda-forge `rdkit` package is installed, that suggests that the `rdkit` package is missing the required pip installation information.

**Comment by zklaus** [comment]
Indeed, the latest version (2023.09.6) has the required `lib/python3.11/site-packages/rdkit-2023.9.6.dist-info` directory that is missing from the old version that you are pinning here. Generally speaking, pinning everything to specific versions is a bit frowned upon, since then we cannot update packages. Can you relax the pins? Which of them are actually really needed?

**Comment by PetroPavliukk** [comment]
> Indeed, the latest version (2023.09.6) has the required `lib/python3.11/site-packages/rdkit-2023.9.6.dist-info` directory that is missing from the old version that you are pinning here. Generally speaking, pinning everything to specific versions is a bit frowned upon, since then we cannot update packages. Can you relax the pins? Which of them are actually really needed?

In theory, I can install a new version of rdkit, but there is a problem that for the dataset on which we generated features and trained the models, the single feature calculation function started to work differently than in the version we used and our reproduced prediction results will a little different. But from what I understand, in that case, I'd better write my own function similar to the one in the old version of rdkit and use the latest version of rdkit?

**Comment by zklaus** [comment]
I am not familiar enough with `rdkit` to comment on that. It is fine to skip the `pip check`. In that case, I would not completely remove it, but comment it out and leave a comment in the `meta.yaml` why this is not feasible at the moment.

**Comment by PetroPavliukk** [comment]
i'll try, thanks a lot for your help

**Comment by ogurbych** [comment]
I would like to be the reviewer for this package

**Comment by DkraAI** [comment]
I would like to be the reviewer for this package.

**Comment by PetroPavliukk** [comment]
@conda-forge/help-python, ready for review!

File: recipes/fluoriclogppka/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    - dgl
+    - dgllife ==0.3.2
+
+test:
+  imports:
+    - fluoriclogppka
+  commands:
+    - pip check || true  # We allow the check to fail because of a problem with rdkit, because the rdkit package is missing the required pip installation information.
+  requires:
+    - pip
```
Comment (BastianZim) [inline recipes/fluoriclogppka/meta.yaml:52]:
```suggestion
```

**Review by BastianZim — APPROVED** [review]

File: recipes/fluoriclogppka/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipes/fluoriclogppka/meta.yaml:52]: Either one works but this is a bit more clean.

**Pull request merged**